docs/data.md

```diff
@@ -183,25 +183,26 @@ mkdir -p /scratch/$LSB_JOBID/grassdata/albers
 # Link the PERMANENT mapset of interest to the temporary file directory using a symlink
 ln -s /rsstu/users/r/rkmeente/FUTURES/grassdata/albers/PERMANENT/ /scratch/$LSB_JOBID/grassdata/albers/
 
-# The resulting directory tree should look something like this:
-tree /scratch/$LSB_JOBID/grassdata/
-/scratch/65834/grassdata/
-└── albers
-    └── PERMANENT -> /rsstu/users/r/rkmeente/FUTURES/grassdata/albers/PERMANENT/
-
-#########   GRASS CODE   #########
 # Run some GRASS code, e.g. a python script:
 grass --tmp-mapset /scratch/$LSB_JOBID/grassdata/albers --exec python script.py 
 
-
-#########   CODE TO REMOVE TEMPORARY FILE DIRECTORY   #########
 # Remove the temporary files and directory
 rm -fr /scratch/$LSB_JOBID
 ```
 
+The resulting directory tree should look something like this:  
+```tcsh
+$ tree /scratch/$LSB_JOBID/grassdata/
+/scratch/65834/grassdata/
+└── albers
+    └── PERMANENT -> /rsstu/users/r/rkmeente/FUTURES/grassdata/albers/PERMANENT/
+```
+Note that `tree /scratch/$LSB_JOBID/grassdata/` will need to be executed in the above script after linking the PERMANENT mapset.  
```

Comment:
will -> would: I didn't think about it before, but I'm afraid this may create another confusion. Something needs to suggest here that to get this output, you need to run it in the script, but you don't have to run it in the script.

Reply:
Good point. Let me know what you think now!

Reply:
As clear as it can be. Thanks.